packages/gravity-ui-web/src/sass/06-utilities/_padding.scss

```diff
@@ -0,0 +1,11 @@
+$grav-sides: (top, bottom, left, right);
+$grav-spaces-map: ('xs': $grav-sp-s,'s': $grav-sp-s, 'm': $grav-sp-m, 'l': $grav-sp-l, 'xl': $grav-sp-xl, 'xxl': $grav-sp-xl);
+
+// Generate utility classes for available spacing sizes
+@each $name, $space in $grav-spaces-map {
+  @each $side in $grav-sides {
+    .grav-u-p#{str-slice($side, 0, 1)}-#{$name} {
+      padding-#{$side}: #{$space} !important;
```

Comment:
I noticed we're not really using it on utility classes but it's usually convention to add it to your trumps. You want to guarantee that it will win or else the class shouldn't be used. Happy to remove it though.

Reply:
I did a bit of digging around and it does seem common practice to have `!important` on utilities - they should be used to enforce a single rule but you wouldn't want that rule to be overridden or else the class should be removed from the HTML. But if you think that's not in line with Gravity's standards then I can remove.

Reply:
No it's OK. Makes sense in this instance.

Did you run the sass linter? If it flags up the use of `!important` you may need to add a `// stylelint-disable-next-line ... ` comment to exclude this instance.